### PR TITLE
Feature/level editor

### PR DESCRIPTION
**Pull request by gwenvis**

### CODE REVIEW

File: Assets/Scripts/Puzzles/Color/LevelEditor/Highlight.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿using UnityEngine;
+
+namespace DN.Puzzle.Color.Editor
+{
+	/// <summary>
+	/// ADD CLASS SUMMARY!
+	/// </summary>
+	public class Highlight : MonoBehaviour
```
Comment (joppevanbasten) [inline Assets/Scripts/Puzzles/Color/LevelEditor/Highlight.cs]: ADD CLASS SUMMARY!

File: Assets/Scripts/Puzzles/Color/LevelEditor/LevelDataDropdown.cs [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+		protected void OnEnable()
+		{
+			dropdown.onValueChanged.AddListener(OnDropdownValueChanged);
+		}
+
+		private void OnDropdownValueChanged(int arg0)
+		{
+			
+		}
```
Comment (joppevanbasten) [inline Assets/Scripts/Puzzles/Color/LevelEditor/LevelDataDropdown.cs]: Why Empty?

File: Assets/Scripts/Puzzles/LevelData.cs [hunk 3]
```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+
+namespace DN
+{
+	/// <summary>
+	/// ye
+	/// </summary>
```
Comment (joppevanbasten) [inline Assets/Scripts/Puzzles/LevelData.cs]: Nice Summary

File: Assets/Scripts/Puzzles/LevelData.cs [hunk 4]
```diff
@@ -0,0 +1,22 @@
+﻿using UnityEngine;
+
+namespace DN
```
Comment (joppevanbasten) [inline Assets/Scripts/Puzzles/LevelData.cs:3]: Namespace gewoon DN?

Comment (gwnvisje) [reply Assets/Scripts/Puzzles/LevelData.cs:3]: yeah

**Review by FloydMa1 — APPROVED** [review]
ANOTN YOU SO GOOD!

**Pull request merged**